README.md

```diff
@@ -240,9 +240,18 @@ pip3 install --pre torch torchvision --extra-index-url https://download.pytorch.
 
 and functorch
 ```
+pip3 install ninja  # Makes the build go faster
 pip install "git+https://github.com/pytorch/functorch.git"
 ```
 
+If this fails, you can get the latest version of functorch that was marked to be 
+compatible with the current torch version:
+```bash
+pip3 install ninja  # Makes the build go faster
+PYTORCH_VERSION=`python -c "import torch.version; print(torch.version.git_version)"`
+pip install "git+https://github.com/pytorch/pytorch.git@$PYTORCH_VERSION#subdirectory=functorch"
```

Comment:
Ohh this is sweet!

Would it also work for the older functorch (0.1.0) ?

Reply:
so basically it's gonna install the functorch that corresponds to your pytorch (so you don't pick up the functorch version but the pytorch version picks up the functorch version -- this is an ancient Chinese proverb by the way)

I'm not sure if that will work with functorch versions that are prior to functorch migration to pytorch/pytorch though